eZ/Publish/Core/settings/repository.yml

```diff
@@ -71,3 +71,7 @@ services:
     ezpublish.api.service.user_preference:
         alias: eZ\Publish\Core\Event\UserPreferenceService
         public: true
+
+    ezpublish.api.service.setting:
```

Comment:
Not a huge fan of this. ezpublish no longer exists, so this just adds to already large technical debt. I prefer to have to refactor less than more at once in the future